api/v1beta1/common.go

```diff
@@ -29,7 +30,7 @@ type extraConfigmapsMounts struct {
 
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// +kubebuilder:validation:optional
-	// +kubebuilder:default=""
+	// +kubebuilder:default:="/"
```

Comment:
This should stay `""`. More info here:

```
// Path within the volume from which the container's volume should be mounted.
// Defaults to "" (volume's root).
// +optional
SubPath [string](https://pkg.go.dev/builtin#string) `json:"subPath,omitempty" protobuf:"bytes,4,opt,name=subPath"`
```

[1] https://pkg.go.dev/k8s.io/api/core/v1#VolumeMount